lib/ae_mdw_web/websocket/subscriptions.ex

```diff
@@ -73,6 +73,27 @@ defmodule AeMdwWeb.Websocket.Subscriptions do
     end
   end
 
+  @spec has_subscribers(version(), channel) :: boolean()
+  def has_subscribers(version, channel) do
+    {:ok, channel_key} = channel_key(version, channel)
+
+    case :ets.match(@subs_channel_pid, {channel_key, :"$1"}, 1) do
+      {_some, _continuation} -> true
+      @eot -> false
+    end
+  end
+
+  @spec has_object_subscribers(version()) :: boolean()
+  def has_object_subscribers(version) do
```

Comment:
Can we add `?` to the end of the function names for these?

Reply:
Done